CMakeLists.txt

```diff
@@ -567,7 +567,7 @@ endif (EXISTS "${PROJECT_SOURCE_DIR}/.svn")
 
```

Comment:
Reminder to add to the PR description the part about the new offset field.

Reply:
Do you think the instr_t.offset portion should be split into its own PR?  I was leaning that way -- or do you think it's fine all as one (less work; but less isolation + harder rollback if it breaks sthg later)?

Reply:
I'd be fine keeping it as one. This PR is not too big. In a different environment I'd suggest splitting (where we're doing regular prod deployments and want to wait between major changes), no need to split to just submit as two parts (unless we want to track both works separately). If we're rolling back, we'd have to roll back both parts anyway.

Reply:
Ack.

Reply:
Updated the PR description above.